Mono.Debugger.Soft/Mono.Debugger.Soft/EventRequest.cs

```diff
@@ -34,6 +34,11 @@ public abstract class EventRequest {
 			}
 		}
 
+		public int getId ()
```

Comment:
Just codestyle: the property must be called with capital first letter

Reply:
Done.